app/Models/Signup.php

```diff
@@ -141,6 +141,7 @@ public function toBlinkPayload()
             'campaign_run_id' => (string) $this->campaign_run_id,
             'quantity' => $quantity,
             'why_participated' => $this->why_participated,
+            'source_details' => $this->source_details,
```

Comment:
hello @DoSomething/storm @sheyd ! We are adding a new column to the `signups` table called `source_details`. This column will record things like where the signup was created. 

For some context, this came out of [this comment](https://github.com/DoSomething/phoenix/pull/7546/files#r197483566), where we are updating the `source_details` from `phoenix-web` to `phoenix-ashes`. 

We'd like for Quasar and Customer.io to ingest this data - but want to make sure this won't break anything. Can you confirm this will be ok?

Reply:
This won't break any current data pipelines, but we need to update the consumer on Quasar to take into the account for the field updates and update the schema in Quasar. What's the timeline for rolling this out to production?

Reply:
good to know, thanks @sheyd ! I was going to merge once you gave the thumbs up - should I wait until the updates are made on the Quasar side? If it won't break anything, maybe I can merge this work and then wait until the consumer is updated on the Quasar side to run [this script](https://github.com/DoSomething/rogue/blob/master/app/Console/Commands/UpdateSignupAndOrPostField.php) to update the source to `phoenix-ashes` instead of `phoenix-web`?

Reply:
@chloealee I'm working on the Quasar side updates right now. One catch I just talked through with @DFurnes was if this field also exists on `posts`. And it does, but it's not currently being sent to Blink or Quasar. Can we update these methods: https://github.com/DoSomething/rogue/blob/master/app/Models/Post.php#L179-L247 to send those `source_details` along as well with this PR? I'll include those updates on Quasar side as well, and we can "pull the trigger" around the same time :)

Reply:
@sheyd ah yes! Thanks for flagging that - just updated to also send a post's `source_details` to Customer.io and Quasar as well!